packages/common/package.json

```diff
@@ -1,57 +1,28 @@
 {
   "name": "@apollo/react-common",
   "description": "React Apollo common utilities.",
-  "version": "3.1.3",
+  "version": "4.0.0-beta.0",
   "author": "opensource@apollographql.com",
   "keywords": [
     "apollo",
     "graphql",
     "react"
   ],
   "license": "MIT",
-  "main": "./lib/react-common.cjs.js",
-  "module": "./lib/react-common.esm.js",
-  "typings": "./lib/index.d.ts",
+  "main": "index.js",
+  "types": "index.d.ts",
   "repository": {
     "type": "git",
     "url": "apollographql/react-apollo"
   },
   "sideEffects": false,
   "scripts": {
-    "clean": "rm -Rf ./lib/* ./meta/bundlesize/* ./meta/coverage/* ./.rpt2_cache",
-    "prepare": "npm run build",
-    "prebuild": "npm run clean",
-    "build": "npx tsc -p ./config",
-    "postbuild": "npx rollup -c ./config/rollup.config.js",
-    "watch": "npx tsc-watch --onSuccess \"npm run postbuild\" -p ./config",
-    "predeploy": "npm run build",
-    "deploy": "npm publish",
-    "test": "npx jest --config ../../config/jest.config.js --testPathPattern packages/common",
-    "test:watch": "npx jest --config ../../config/jest.config.js --testPathPattern packages/common --watch",
-    "test:cjs": "npm run build && npx jest --config ../../config/jest.cjs.config.js --testPathPattern packages/common",
-    "test:umd": "npm run build && npx jest --config ../../config/jest.umd.config.js --testPathPattern packages/common"
-  },
-  "peerDependencies": {
-    "@types/react": "^16.8.0",
-    "apollo-client": "^2.6.4",
-    "apollo-utilities": "^1.3.2",
-    "graphql": "^14.3.1",
-    "react": "^16.8.0"
+    "clean": ""
```

Comment:
Does this still need these empty `clean` scripts?

Reply:
Without these `lerna` complains when we run `npm run clean` from the monorepo root. We could adjust lerna to silence its complaints, or handle multi package cleaning without lerna, but leaving them as empty script references was the easiest. Happy to adjust further if you think we should. Thanks!